templates/2_create_tiled_mibi_run.ipynb

```diff
@@ -13,12 +13,14 @@
    "cell_type": "markdown",
```

Comment:
<p>this can be one cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>